src/components/CallView/CallView.vue

```diff
@@ -75,17 +75,22 @@
 							:shared-data="sharedDatas[shownRemoteScreenPeerId]"
 							is-big />
 						<!-- presenter overlay -->
-						<TransitionWrapper v-if="shouldShowPresenterOverlay"
-							name="slide-down">
+						<VueDraggableResizable v-if="shouldShowPresenterOverlay"
+							parent
+							:resizable="false"
+							:h="overlaySize"
+							:w="overlaySize"
+							:x="10"
+							:y="10">
 							<VideoVue class="presenter-overlay__video"
 								:token="token"
 								:model="shownRemoteScreenCallParticipantModel"
 								:shared-data="sharedDatas[shownRemoteScreenPeerId]"
 								is-presenter-overlay
 								un-selectable
 								hide-bottom-bar
-								@click-video="toggleShowPresenterOverlay" />
-						</TransitionWrapper>
+								@click-presenter="toggleShowPresenterOverlay" />
```

Comment:
That's because screenshare overwrites video stream if present ( by order ). 
I think it's not possible even from the stripe ( click on the video stream just promote the screenshare).

Reply:
Ref in https://github.com/nextcloud/spreed/issues/11789